kalliope/core/Utils/FileManager.py

```diff
@@ -37,11 +37,10 @@ def write_in_file(file_path, content):
         """
         try:
```

Comment:
And update doc for the method with : 
`
 :param content: the contents to write in the file
 :type content: str
`